src/main/scala/ie/zalando/fabric/gateway/web/marshalling/JsonModels.scala

```diff
@@ -141,18 +141,29 @@ trait JsonModels {
       })
     }
 
+  implicit val decodeStaticRouteConfig: Decoder[StaticRouteConfig] = (c: HCursor) =>
+    for {
+      statusCode    <- c.downField("status").as[Int]
```

Comment:
Nicer to keep the `<-` inline? Minor nit

Reply:
Yeah I will run scalafmt after the pr review is done, it's a lot of changes otherwise